lib/handlers/text.js

```diff
@@ -12,6 +12,17 @@ import {u} from 'unist-builder'
 export function text(h, node) {
   return h.augment(
     node,
-    u('text', String(node.value).replace(/[ \t]*(\r?\n|\r)[ \t]*/g, '$1'))
+    u(
+      'text',
+      String(node.value)
+        .replace(/^[ \t]+/gm, '')
+        .split('')
+        .reverse()
+        .join('')
+        .replace(/^[ \t]+/gm, '')
+        .split('')
+        .reverse()
+        .join('')
+    )
```

Comment:
```suggestion
    u('text', trimLines(node.value))
```

Could you switch this PR to use `trim-lines` from `npm` as a dependency?

Reply:
Yes, I think it makes sense to utilize an equivalent library that can be independently maintained.